content/install-ubuntu.md

```diff
@@ -55,137 +53,143 @@ Ubuntu may need a special option enabled to install and boot from Ubuntu until t
 
 From the menu shown the 'nomodeset' option can be selected during this boot. -->
 
-![grub](/images/install-ubuntu/ubuntu-grub.png)
+![grub](/images/install-ubuntu/GRUB-2.12.png)
 
-Once Ubuntu starts, you will be asked to **Try Ubuntu without installing** or to **Install Ubuntu**. If you are attempting to restore a broken operating system, please choose **Try Ubuntu**.
+Once Ubuntu starts and you have selected your language, any accessibility features that you may need, your keyboard selection, and network connectivity you will be asked to **Try Ubuntu without installing** or to **Install Ubuntu**. If you are attempting to restore a broken operating system, please choose **Try Ubuntu**.
```

Comment:
Can we split the first sentence here so it's not so long? (Otherwise, you would need a comma after "network connectivity.")

> Once Ubuntu starts, you will be asked to select your language, any accessibility features that you may need, your keyboard layout, and network connectivity. Next, you will be asked to **Try Ubuntu without installing** or to **Install Ubuntu**.